.github/workflows/tg-nightly-tests-impl.yaml

```
@@ -0,0 +1,70 @@
+name: "[internal] TG nightly tests impl"
+
+on:
+  workflow_call:
+
+jobs:
+  tg-nightly-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        test-group: [
+          { name: "TG CCL tests", arch: wormhole_b0, cmd: run_tg_nightly_ccl_tests, timeout: 40, owner_id: ULMEPM2MA}, # Sean Nijjar
+        ]
+    name: ${{ matrix.test-group.name }}
+    runs-on: ["arch-wormhole_b0", "config-tg", "in-service", "pipeline-functional"]
+    container:
+      image: ${{ needs.build-artifact.outputs.ci-build-docker-image }}
```

In my experience GHA will silently collapse this down to nothing and then not run in a container.  So you won't notice this unless you inspect the build steps of the run itself.

```suggestion
      image: ${{ inputs.docker-image }}
```